notebooks/D_CNN_Inference_and_Embedding.ipynb

```diff
@@ -22,7 +22,11 @@
     "\n",
     "1. Execute the first cell containing code below, which will install the CellX library & create a local test directory in the environment of the virtual machine. The executed first cell will print ```Building wheel for cellx (setup.py) ... done```. (Note: This virtual environment is different from the one created for the Training notebook, which is why we need to re-install the external `cellx` library etc.)\n",
     "\n",
-    "2. Click on the ``` 📁``` folder icon located on the left-side dashboard of the Colab notebook. Drag your annotated zip file(s) into the \"test\" folder and your saved model (the `.h5` file) into the \"content\" folder - the parent folder of the \"test\" folder.\n",
+<<<<<<< HEAD
+    "2. Click on the ``` 📁``` folder icon located on the left-side dashboard of the Colab notebook. Drag your saved model (the `.h5` file) into the `/content/` folder and your annotated zip file(s) into the `/content/test/` folder.\n",
+=======
+    "2. Click on the ``` 📁``` folder icon located on the left-side dashboard of the Colab notebook, this is the default `content` directory where you can see the following subdirectories: `sample_data` (default) & `test`. Drag your saved model (the `.h5` file) into the `content` folder and your annotated zip file(s) into the `test` folder.\n",
+>>>>>>> fix #34 & minor markdown edits
```

Comment:
Hi @laureho - it looks like you still have some git conflict markers here - can you resolve these, then I'll review.

Reply:
sorry about that, should be solved now!